app/styles/components/_footer.scss

```diff
@@ -16,19 +16,41 @@ $icon-size: 25px;
   background-color: $Theme-color-blue;
 }
 
-.Footer-content {
-  @include font-properties(sans, desktop, content, large);
+.Footer-links {
+  @include font-properties(sans, desktop, content, small);
+  
+  display: flex;
+  flex-direction: column;
+}
 
-  margin-top: $Theme-margin-small;
+.Footer-link {
 
+  &:not(:first-child) {
+    margin-top: $Theme-margin-xSmall;
+  }
+
+  text-transform: uppercase;
   color: $Theme-color-white;
   font-weight: bold;
+
+  &:hover {
+    opacity: $Theme-opacity-half;
+  }
```

Comment:
Makes sense to me to affect opacity instead of color, added transition 0.4s to be consistent with topnav;